engine/language-server/src/main/schema/binary_protocol.fbs

```diff
@@ -13,10 +13,10 @@ union InboundPayload {
 // An envelope for inbound requests and commands.
 table InboundMessage {
   // A unique id of the message sent to the server.
-  messageId: EnsoUUID (required);
+  message_id: EnsoUUID (required);
```

Comment:
Looks like can incompatible, unnecessary change. Can we continue to use the old names of identifiers?

Reply:
The newer `flatc` version issue warnings when fields are not following `snake_case`, so we should change unless there is a good reason not to.

The compatibility is not affected. The binary protocol itself is agnostic of field names. In the generated code the `flatc` tries to follow whatever it considers as appropriate for given target language. In our case, the Java code still keeps the `camelCase`.